earth2studio/data/utils.py

```diff
@@ -124,3 +126,50 @@ def prep_data_inputs(
         time = timearray_to_datetime(time)
 
     return time, variable
+
+
+def datasource_to_netcdf(
```

Comment:
Can we make this more general so that users can save as Zarr too?

The main changes might be adding the following keyward arguments
```
...,
backend: Literal["netcdf", "zarr"] = "netcdf",
chunks: dict = {},
)
```

Reply:
Added!